assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss

```diff
@@ -60,7 +60,10 @@
 	}
 
 	.mdc-button {
+		font-weight: $fw-medium;
 		margin: 0 auto;
+		min-height: 32px;
+		padding: 6px 16px;
```

Comment:
Noting that the primary CTA button is the shorter version, it follows the tertiary "Got it" CTA should also be, therefore this styling can be applied across the board to `.mdc_button` within the component (hence also the VRT update, which seems fine as we didn't have a mobile design for reference in the first place (see [Figma](https://www.figma.com/design/7ba0pj1rLuvLvJhy3NiHOj/AdSense?node-id=100-2626&t=wcYZG4atKxLUT3pB-0), theres's no meaningful difference to the wider viewport versions, and the new bottom margin _is_ aligned with the Figma design for the mobile version of the [Audience Segmentation success banner](https://www.figma.com/design/7pSrkEy8t00BcYRAi9LjjH/Audience-Segmentation?node-id=1711-17131&t=nBgPqJ7Y8YIAsiwR-0)).

Note that both buttons are also `font-weight: 500`, the "Got it" tertiary CTA already having this weight.